src/onboarding/onCreateScript.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Manually run the onCreateScript() function first after deployment, ONLY ONCE
+ * to initiate properties, triggers, and others that are needed during running
+ */
+
+function onCreateScript() {
+  createTrigger();
+  
+  setProperty("PROFILE_IMAGE_FILE_ID", PROFILE_IMAGE_FILE_ID, PROPERTIES_TYPE_SCRIPT);
```

Comment:
OK, if this is only once then set the constant in the same file

Reply:
I used to write them in this file. But it looks ugly, so I put the constants to an individual file.